### PR TITLE
Subscriptions: convert link in server message into html link

### PR DESCRIPTION
**Pull request by dperpeet**
We only convert the first link, as more don't seem necessary in our use cases.

### CODE REVIEW

**Comment by dperpeet** [comment]
I copied shell.escape since it doesn't seem worth pulling in the shell package just for this.

**Comment by dperpeet** [comment]
Screenshot:
register_system

**Comment by stefwalter** [comment]
>  I copied shell.escape since it doesn't seem worth pulling in the shell package just for this.

You're right about shell.escape() being an anti-pattern, we should avoid it for most cases. It's similar to php's mysql_real_escape_string().

I would suggest generating HTML nodes directly after your HTML regex, rather than creating an HTML string which you then parse.

**Comment by dperpeet** [comment]
> I would suggest generating HTML nodes directly after your HTML regex, rather than creating an HTML string which you then parse.

Done, escaping is now done via Mustache and only on the non-link parts.

File: pkg/subscriptions/subscriptions.js [hunk 1]
```diff
@@ -369,6 +369,26 @@ define([
             update_timeout = window.setTimeout(status_update_failed, 10000);
         }
 
+        function linkify_message(message) {
+            /* expression from on https://github.com/thp/libcontentaction/blob/47350706ecf4290a74245ae11ef76c4eedde729d/data/gen-regexps #
```
Comment (stefwalter) [inline pkg/subscriptions/subscriptions.js]: The question will come up at some point about license. There's no license in that other file. The project itself is 
```Copyright (c) 2009, 2010 Nokia Corporation.``` even though it looks like @mvollmer wrote it.

https://github.com/thp/libcontentaction/blob/47350706ecf4290a74245ae11ef76c4eedde729d/debian/copyright

**Comment by stefwalter** [comment]
Other than the license issue, this looks good to me.

**Comment by dperpeet** [comment]
good catch. The library it is forked from is LGPL. To avoid all of this I wrote my own regexp, which is also simpler (reflecting our simple use case).

**Comment by dperpeet** [comment]
Additionally I fixed the alignment of the updating status message - it is now centered like similar messages.
subscriptions_centered

**Pull request closed without merging**